src/grep.js

```diff
@@ -22,12 +24,15 @@ common.register('grep', _grep, {
 //@ + `-l`: Print only filenames of matching files.
 //@ + `-i`: Ignore case.
 //@ + `-n`: Print line numbers.
+//@ + `-B <num>`: Show `<num>` lines before each result.
```

Comment:
Do you want to implement `-C` too? Seems like this is the same as `-AB <num>` which you've already implemented.

Reply:
Done!